### PR TITLE
stop using `--buildtype=release` in easyconfigs using `MesonNinja` easyblock

### PR DESCRIPTION
**Pull request by Micket**
Depends on 

- [x] #https://github.com/easybuilders/easybuild-easyblocks/pull/3454

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3454
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/18087124489f6e00fd77d58b05983c3b for a full test report.

**Comment by jfgrimm** [comment]
perhaps worth adding a CI check that catches new easyconfigs that use `--buildtype release` etc?

**Review by jfgrimm — DISMISSED** [review]
lgtm

File: test/easyconfigs/easyconfigs.py [hunk 1]
```diff
@@ -1778,6 +1778,13 @@ def template_easyconfig_test(self, spec):
                 fail_msg += f"'{regex.pattern}'"
                 failing_checks.append(fail_msg)
 
+    # meson buildtype should be specified with easyblock parameter "buildtype" not with custom configopts.
+    if ec['easyblock'] == 'MesonNinja':
+        configopts = ec.get('configopts', '', resolve=True)
+        if '--buildtype ' in configopts:
```
Comment (boegel) [inline test/easyconfigs/easyconfigs.py:1784]: @Micket Should also consider `--buildtype=`?
```suggestion
        if '--buildtype ' in configopts or '--buildtype=' in configopts:

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 58 out of 58 (58 easyconfigs in total)
node3508.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.6.8
See https://gist.github.com/boegel/dd87c0fa03817aa050c790792b06f310 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**